lib/libc/string/wcscasecmp.c

```diff
@@ -41,7 +41,7 @@ wcscasecmp(const wchar_t *s1, const wchar_t *s2)
 		c1 = towlower(*s1);
 		c2 = towlower(*s2);
 		if (c1 != c2)
-			return ((int)c1 - c2);
```

Comment:
Some architectures have wchar_t as int, not unsigned int, and so are still identical to the original code here. Presumably to be 100% free from UB this should be performing the arithmetic on unsigned ints and then (implicitly?) casting back to int.